packages/blocks/src/api/raw-handling/index.js

```diff
@@ -79,7 +80,7 @@ function getRawTransformations() {
  */
 export default function rawHandler( { HTML = '', plainText = '', mode = 'AUTO', tagName, canUserUseUnfilteredHTML = false } ) {
 	// First of all, strip any meta tags.
-	HTML = HTML.replace( /<meta[^>]+>/, '' );
+	// HTML = HTML.replace( /<meta[^>]+>/, '' );
```

Comment:
I did imagine it would affect the following pieces, but wanted to try this anyway. The tests still pass, so I think we need both comments and also some new tests that fail if this line is commented. Would you have some?

Reply:
Sure, I'll have a look.

Reply:
Pushed.